content/events/workshop-amsterdam-2020-03-30.md

```diff
@@ -0,0 +1,55 @@
+---
+title: KubeCon EU  Pre-day Workshop
+url: /events/workshop-amsterdam-2020-03-30
+type: page
+layout: event-single
+calendly_url: "https://calendly.com/pulumi/kubecon?month=2020-03&hide_event_type_details=1&primary_color=512668"
```

Comment:
Should we just put the `&hide_event_type_details=1&primary_color=512668` part of the URL in the template itself? It seems kinda error prone if we expect the `calendly_url` in the page's front mater to have the same formatting, etc.

Reply:
👍